pkg/epinio/models/epinio-resource.js

```diff
@@ -49,6 +50,7 @@ export default class EpinioResource extends Resource {
     opt.method = 'delete';
 
     try {
+      await this.bulkRemove(opt);
```

Comment:
This applies to ALL epinio resources, rather than just the three that support the bulk remove api